.rubocop.yml

```diff
@@ -30,6 +30,8 @@ RSpec/NestedGroups:
   Enabled: false
 Rails/SaveBang:
   Enabled: false
+Rails/UniqueValidationWithoutIndex:
```

Comment:
This was causing rubocop to crash since it doesn't understand rails apps that use sequel :face_with_head_bandage: